README.md

```diff
@@ -81,6 +114,8 @@ Verify that the `asciidoctor-pdf` command is available on your `PATH` by running
 
     $ asciidoctor-pdf --version
 
+**NOTE:** If you get an error about [Executions Policies](https://go.microsoft.com/fwlink/?LinkID=135170) when running this command on PowerShell, try to use the following command instead: `$ asciidoctor-pdf.cmd --version`.
```

Comment:
@rdmueller When using `nvm` to install `npm` and `node`, I got an error about executions policies. The command `asciidoctor-pdf --version` will in fact execute `asciidoctor-pdf.ps1 --version` and for whatever reason executing `ps1` file is not allowed in PowerShell on my machine.

Anyway, with the updated install command it should work: `npm i -g @asciidoctor/core asciidoctor-pdf`

Reply:
I just also succeeded with 
```
    npm i -g @asciidoctor/core asciidoctor-pdf
```

the execution of `ps1` files wasn't a problem for me, because I already removed this restriction a long time ago :-)